src/components/Widgets/MarkdownControl.js

```diff
@@ -35,37 +33,31 @@ class MarkdownControl extends React.Component {
   render() {
     const { onChange, onAddAsset, onRemoveAsset, getAsset, value } = this.props;
     const { mode } = this.state;
-    if (mode === 'visual') {
-      return (
-        <div className="cms-editor-visual">
-          <VisualEditor
-            onChange={onChange}
-            onAddAsset={onAddAsset}
-            onRemoveAsset={onRemoveAsset}
-            onMode={this.handleMode}
-            getAsset={getAsset}
-            value={value}
-          />
-        </div>
-      );
-    }
-
-    return (
+    const visualEditor = (
+      <div className="cms-editor-visual">
+        <VisualEditor
+          onChange={onChange}
+          onAddAsset={onAddAsset}
+          onRemoveAsset={onRemoveAsset}
+          onMode={this.handleMode}
+          getAsset={getAsset}
+          value={value}
+        />
+      </div>
+    );
+    const rawEditor = (
       <div className="cms-editor-raw">
         <RawEditor
           onChange={onChange}
           onAddAsset={onAddAsset}
           onRemoveAsset={onRemoveAsset}
           onMode={this.handleMode}
+          rawMode
```

Comment:
I think we should remove this prop from the `RawEditor` (not `Toolbar`). This is currently the only place this component is initialized, and it's hard-coded to `true`. In addition, this prop is duplicated in the `Toolbar` props within `RawEditor.render`, so it would currently always be true even if this prop *was* set to something else. I can't think of any situation where the `RawEditor` would need to set the toolbar to visual mode.